Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
     crass (1.0.4)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.7.1)
-    i18n (1.1.1)
+    erubi (1.8.0)
+    i18n (1.6.0)
```

Comment:
### [i18n](http://github.com/ruby-i18n/i18n)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.1.1 → 1.6.0

[_[change-log](https://github.com/svenfuchs/i18n/releases), [source-code](https://github.com/svenfuchs/i18n)_]





<details>
<summary>Commits</summary>

A change of **88** commits. See the full changes on [the compare page](https://github.com/ruby-i18n/i18n/compare/v1.1.1...v1.6.0).

These are the first 10 commits:
- (15343c5) [Provide a uniform api between Simple, KeyValue and Chain sto…](https://github.com/ruby-i18n/i18n/commit/15343c5935ce904ab91daa94871918b62b8b7134)
- (4260c62) [support translation hashes with numeric keys in Simple backe…](https://github.com/ruby-i18n/i18n/commit/4260c62cf628d60bd9fd45dd1315629c653cd8ab)
- (1092f6c) [add CacheFile backend module](https://github.com/ruby-i18n/i18n/commit/1092f6c6f480d7df20834ce769b35d12f85a4c2e)
- (ebcd95a) [Support JSON as a translation file format.](https://github.com/ruby-i18n/i18n/commit/ebcd95ac57254bc2dbb83991ceca88ff0d223bcf)
- (1e1a3bc) [Fix JSON gem dependency.](https://github.com/ruby-i18n/i18n/commit/1e1a3bcc15de777b42847262b630aef77f569e58)
- (ed9ceb0) [Add links to additional wikis/documentation](https://github.com/ruby-i18n/i18n/commit/ed9ceb086fd3995c928b68a27077e7023f95c4c1)
- (6c09164) [Add Rails 5.2 Gemfile](https://github.com/ruby-i18n/i18n/commit/6c0916474ada85652ac0050d5df6c7ff7a819fbf)
- (120442f) [Merge pull request #429 from violetatrium/master](https://github.com/ruby-i18n/i18n/commit/120442f4322c978d088ddadf06657a78aa68cf1d)
- (8f2d961) [Merge pull request #422 from wjordan/simple_numeric_keys](https://github.com/ruby-i18n/i18n/commit/8f2d961e7feccf8e2a7a7e53e646ef6930ea54a5)
- (40bca10) [Merge pull request #423 from wjordan/backend_cache_file](https://github.com/ruby-i18n/i18n/commit/40bca10382b555afcd378cbe7cb02d7f5f977864)


</details>